.github/workflows/ci.yml

```
@@ -27,75 +41,171 @@
+          flutter pub get || (flutter pub upgrade --major-versions && flutter pub get)
+
+      - name: Generate Localization Files
+        run: flutter gen-l10n
 
       - name: Build Runner (Generate Dart Files)
         run: flutter packages pub run build_runner build --delete-conflicting-outputs
 
-      - name: Format Check (Exclude .mocks files)
-        run: dart format --set-exit-if-changed $(git ls-files '*.dart' ':!:*.mocks.dart')
+      - name: Format Check (Exclude generated files)
+        if: matrix.build-type != 'windows'
```

actionlint (1.7.7)

property "build-type" is not defined in object type {platform: string}

(expression)